src/person/utils.py

```diff
@@ -0,0 +1,21 @@
+from os import environ
+
+from api.utils import failure_response
+from django.contrib.auth.models import User
+import requests
+
+
+def upload_profile_pic(user_id, profile_pic_base64):
+    """Uploads image to server and modifies user's profile_pic_url"""
+    request_body = {
+        "bucket": environ.get("IMAGE_BUCKET_NAME"),
+        "image": profile_pic_base64,
+    }
+    response = requests.post(environ.get("IMAGE_UPLOAD_URL"), json=request_body).json()
```

Comment:
you should be importing from `settings.py`, and your `settings.py` should be importing from `os.environ.get()`